README.md

```diff
@@ -15,251 +15,191 @@
+```java
+public class MessageReceiveListener extends ListenerAdapter {
+  @Override
+  public void onMessageReceived(MessageReceivedEvent event) {
+    System.out.printf("[%s] %#s: %s\n",
+      event.getChannel(),
+      event.getAuthor(),
+      event.getMessage().getContentDisplay());
+  }
+}
+```
 
-By default, the **InterfacedEventManager** is used.
-Since you can create your own implementation of `IEventManager`, this is a very versatile and configurable system.
-If the aforementioned implementations don't suit your use-case you can simply create a custom implementation and
-configure it on the `JDABuilder` with `setEventManager(...)`.
+You can find a more thorough example with the [MessageLoggerExample](https://github.com/discord-jda/JDA/blob/master/src/examples/java/MessageLoggerExample.java) class.
 
-#### Examples:
+### Example: Slash Command Bot
 
-**Using EventListener**:
+This is a bot that makes use of [interactions](https://jda.wiki/using-jda/interactions/) to respond to user commands. Unlike the message logging bot, this bot can work without any enabled intents, since interactions are always available.
 
 ```java
-public class ReadyListener implements EventListener {
-    public static void main(String[] args) throws InterruptedException {
-        // Note: It is important to register your ReadyListener before building
-        JDA jda = JDABuilder.createDefault("token")
-            .addEventListeners(new ReadyListener())
-            .build();
-
-        // optionally block until JDA is ready
-        jda.awaitReady();
-    }
+public static void main(String[] args) {
+  JDA jda = JDABuilder.createLight(token, Collections.emptyList())
+      .addEventListener(new SlashCommandListener())
+      .build();
 
-    @Override
-    public void onEvent(GenericEvent event) {
-        if (event instanceof ReadyEvent) {
-            System.out.println("API is ready!");
-        }
-    }
-}
-```
+  // Register your commands to make them visible globally on Discord:
 
-**Using ListenerAdapter**:
+  CommandListUpdateAction commands = jda.updateCommands();
 
-```java
-public class MessageListener extends ListenerAdapter {
-    public static void main(String[] args) {
-        JDA jda = JDABuilder.createDefault("token")
-                .enableIntents(GatewayIntent.MESSAGE_CONTENT) // enables explicit access to message.getContentDisplay()
-                .build();
-        //You can also add event listeners to the already built JDA instance
-        // Note that some events may not be received if the listener is added after calling build()
-        // This includes events such as the ReadyEvent
-        jda.addEventListener(new MessageListener());
-    }
+  // Add all your commands on this action instance
+  commands.addCommands(
+    Commands.slash("say", "Makes the bot say what you tell it to")
+      .addOption(STRING, "content", "What the bot should say", true) // Accepting a user input
+  );
```

Comment:
Is there any reason to not just combine these into a single chain? it likely would make it feel nicer to read